unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsHUD/SettingsHUDView.cs

```diff
@@ -6,6 +6,8 @@ public class SettingsHUDView : MonoBehaviour
     {
         public bool isOpen { get; private set; }
 
+        public event System.Action OnClose;
```

Comment:
Is it possible to move this event to the controller?

Reply:
For now we are following the same resposibility principle than other views (as I have told in the previous comment, we can discuss it).

Reply:
I think Pato may be onto something here. In the rest of our views, the events are triggered by UI interactions, i.e. `OnClose` would mean the user clicked on the close button. Here, the event is triggered by a state change, and the state owner is the controller.

Reply:
Ok, I will take a look to it, thanks guys.

Reply:
Done!